models/UserSocialMediaModel.ts

```diff
@@ -0,0 +1,29 @@
+import { BaseEntity, Column, Entity, Index, PrimaryGeneratedColumn, JoinColumn, ManyToOne } from 'typeorm';
+import { UserModel } from './UserModel';
+import { Uuid, SocialMediaType, PublicUserSocialMedia } from '../types';
+
+@Entity('UserSocialMedias')
+export class UserSocialMediaModel extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid')
+  uuid: Uuid;
+
+  @ManyToOne((type) => UserModel, (user) => user.feedback, { onDelete: 'CASCADE' })
+  @JoinColumn({ name: 'user' })
+  @Index({ unique: true })
+  user: UserModel;
```

Comment:
Should this column have a unique constraint? I think if a user has multiple social media links on their profile, then there would be more than one `UserSocialMediaModel`s with the same user, which should be legal according to our business logic.